acceptance/windows-service/test/integration/chef-windows-service/inspec/chef_windows_service_spec.rb

```diff
@@ -0,0 +1,58 @@
+only_if do
+  os['family'] == 'windows'
+end
+
+describe command("chef-service-manager") do
+  it { should exist }
+  its('exit_status') { should eq 0 }
+end
+
+describe service('chef-client') do
+  it { should_not be_enabled }
+  it { should_not be_installed }
+  it { should_not be_running }
+end
+
+describe command("chef-service-manager -a install") do
```

Comment:
Maybe this is just something new that I'll need to get used to, but it seems weird to be modifying the system-under-test inside the test.  Anyone else disagree?  Or is it okay because its only running the service manager?